README.md

```diff
@@ -204,7 +205,7 @@ The UI class contains static methods for displaying the list of books in the HTM
 
 Together, these classes allow the user to add books to a list, remove books from the list, and have the list persist even after the page is refreshed.
   
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>..
```

Comment:
- [ ] it would be better to make sure that the `MIT` link is working and it should take you to the License of the project in order for your project will be professional